support/reftest/reftest

```diff
@@ -38,7 +38,9 @@ class DBHandler:
     ):
         if not from_date:
             from_date = self.default_from_date
-        return self.dynamodb.put_item(
+        if not content_type:
+            content_type = ""
```

Comment:
If the intent here was to cover what happens when there's no content_type, I don't think this quite achieves that, does it?  This will store content_type of `""` (empty string) on the created items.

Most items in the DB right now have *no* content_type at all, which is slightly different.  Should you do the same here?